js/apps/admin-ui/src/clients/keys/GenerateKeyDialog.tsx

```diff
@@ -99,6 +99,7 @@ export const KeyForm = ({
               variant={SelectVariant.single}
               aria-label={t("archiveFormat")}
               isOpen={openArchiveFormat}
+              menuAppendTo="parent"
```

Comment:
The options in the dropdown were clipping on the bounds of the modal dialog, this helps prevent that.